governance/polls/Ratification Poll for Adding Sidestream Auction Services Core Unit, SAS-001 - November 8, 2021.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Ratification Poll for Adding Sidestream Auction Services Core Unit, SAS-001 - November 8, 2021
+summary: MIP39c2-SP25 adds SAS-001: Sidestream Auction Services Core Unit.
+discussion_link: https://forum.makerdao.com/t/mip39c2-sp25-adding-sidestream-auction-services-core-unit-sas-001/10796
+vote_type: Plurality Voting
+categories:
+   - MIPs
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2021-11-08T16:00:00
+end_date: 2021-11-22T16:00:00
+---
+# Inclusion Poll for Adding Sidestream Auction Services Core Unit, SAS-001 - November 8, 2021
+
+The Governance Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, November 8 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to $mip_set being accepted and implemented in the Maker Protocol.
+
+### MIP39c2-SP25
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*The SAS-001 Core Unit aims to improve security, transparency and accessibility of the Maker Protocol by providing and maintaining auction services through open-source development.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Cannonical Proposal Version](https://github.com/makerdao/mips/blob/1e68595057de5e9d2154a3320ff121ea7ad1f94a/MIP39/MIP39c2-Subproposals/MIP39c2-SP25.md)
+* [Latest Proposal Version](https://mips.makerdao.com/mips/details/MIP39c2SP25)
+* [Proposal Discussion Thread](https://forum.makerdao.com/t/mip39c2-sp25-adding-sidestream-auction-services-core-unit-sas-001/10796)
+
+### MIP40c3-SP44
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*MIP40c3-SP44 adds the MKR Incentive Plan budget for SAS-001: Sidestream Auction Services Core Unit. It contains:*
```

Comment:
Reformat. Currently it looks like this:

![image](https://user-images.githubusercontent.com/72349134/140408219-2050c62b-8574-4954-9199-e81bbbf381c9.png)